executor/mock/standalone/standalone_test.go

```diff
@@ -875,3 +876,28 @@ func testTtyNegative(t *testing.T, jobID string) {
 		t.Fail()
 	}
 }
+
+func testCachedDockerPull(t *testing.T, jobID string) {
+
+	var imgName = "nginx"
```

Comment:
Is there a titusops image worth pulling that's small, so we can ensure it's not reused? 

Also, it's probably worth deleting the image before it's pulled the first time, because people run this test with long-running VMs.